src/runtime/profiler.cpp

```diff
@@ -331,44 +335,50 @@ WEAK void halide_profiler_report_unlocked(void *user_context, halide_profiler_st
 
                 sstr << "  " << fs->name << ": ";
                 cursor += 25;
-                while (sstr.size() < cursor) sstr << " ";
+                while (sstr.size() < cursor)
```

Comment:
while loops in this function now need braces

Reply:
Done